### DISCUSSION TITLE
I downloaded the ark library, but the segmentation module is not there?

### DISCUSSION THREAD

**Discussion by mmhatmal** [angelolab/ark-analysis#1002] [General]
Hi, I downloaded the ark library, but the segmentation module is not there?, the following code in pycharm (jetbrains):
"from ark.segmentation import marker_quantification, segmentation_utils" gives a red mistake line under segmentation, marker_quantification, and segmentation_utils.

the error message says "Cannot find reference 'segmentation' in '__init__.py'"

any help please?

**Comment by srivarra** [comment]
Hi, I was wondering if you have installed ark with `pip install .` if you are in the cloned repo, or `pip install ark-analysis`? You can get a list of your currently installed packages with `pip list`.
